### DISCUSSION TITLE
Condition should hide field from filter and grid both

### DISCUSSION THREAD

**Discussion by BhargavSuthar99** [payloadcms/payload#7389] [Feature Requests & Ideas]
### Payload Version

3.0.0-beta.43 and 3.0.0-beta.68

### Node Version

20.12.12

### Next.js Version

15.0.0-canary.53

### Describe the Bug

Describe the Bug
Condition function should hide field from column and filter.
Or
Convert disableListFilter and disableListColumn properties to function for hide those fields.

### Reproduction Steps

To Reproduce
Open collection page
Make access parameters like read: () => true or false, update: () => false, create: () => false,
Write condition functions to condition: (_, siblingData) => false,
Field is now hidden from page but still visible on collection column and filter page.

https://github.com/user-attachments/assets/f591c2a0-96cd-499f-aa15-6f1d01f1a752

**Comment by paulpopus** [comment]
Converting this to a feature request discussion. I like the idea of making `disableListFilter` and `disableListColumn` take functions instead!
